### PR TITLE
feat: implement top-place-section for mobile

### PR DESCRIPTION
**Pull request by quanghuy1799**
task: https://trello.com/c/hWCIGwTm

### CODE REVIEW

File: practice-two/src/css/main.css [hunk 1]
```diff
@@ -34,6 +34,43 @@
...
+}
+.best-location-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  .best-location-card,
+  .best-location-card-title,
+  .best-location-card-text,
+  .rate-star {
+    text-align: unset;
```
Comment (bannguyen-agilityio) [inline practice-two/src/css/main.css:86]: Can you explain why you use unset here?

File: practice-two/src/css/main.css [hunk 2]
```diff
@@ -34,6 +34,43 @@
...
+  }
+  .previous-btn {
+    width: 64px;
+    height: 64px;
+    background: #fff;
+    border-radius: 100%;
+    border: 1px solid rgba(25, 24, 37, 0.5);
+  }
+  .next-btn {
+    width: 64px;
```
Comment (bannguyen-agilityio) [inline practice-two/src/css/main.css]: pre and next buttons should have the same styles. They only differ in its content (icon).

File: practice-two/src/css/main.css [hunk 3]
```diff
@@ -34,6 +34,43 @@
   display: none;
 }
 
+.best-location-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```
Comment (bannguyen-agilityio) [inline practice-two/src/css/main.css:53]: Not sure if we use flex-start here or not. I think the content should be central

File: practice-two/src/index.html [hunk 4]
```diff
@@ -32,6 +32,71 @@ <h1 class="logo-text">Travlog</h1>
...
+          </span>
+          <h2 class="section-heading best-location-heading">
+            Explore top destination
+          </h2>
+          <div class="carousel-button">
+            <button class="previous-btn">
+              <img
+                src="assets/images/previous-icon.png"
+                width="15px"
+                height="15px"
```
Comment (bannguyen-agilityio) [inline practice-two/src/index.html:50]: We should use CSS styling for easier maintenance

File: practice-two/src/css/main.css (hunk 2 above)
Comment (quanghuy1799) [reply practice-two/src/css/main.css]: i will check it

File: practice-two/src/index.html (hunk 4 above)
Comment (quanghuy1799) [reply practice-two/src/index.html:50]: i will fix it!

File: practice-two/src/css/main.css (hunk 3 above)
Comment (quanghuy1799) [reply practice-two/src/css/main.css:53]: i will check it and reply in DM!!!

File: practice-two/src/css/main.css (hunk 1 above)
Comment (quanghuy1799) [reply practice-two/src/css/main.css:86]: because in commond style, i was set text-align: center;

**Pull request closed without merging**